### PR TITLE
Add function to show random data on page load.

### PR DESCRIPTION
**Pull request by mdflynn**
## What is the change?

This add the functionality to generate a random poster, title, and descriptors when the webpage loads.

## Is this a fix or a feature?

This is a feature. It finished iteration 0.

## Where should the reviewer start?

Start with the main.js file and start with line 2. Review all new code added.

## How should this be tested?

Open index.html in your terminal. Refresh the page multiple times to see a randomly generated cover image, titles and set of descriptors.

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -8,12 +11,23 @@ var savedCovers = [
 var currentCover;
 
 // Add your event listeners here 👇
-
+window.onload = displayMainPageCover;
```
Comment (coopterrones) [inline src/main.js:14]: great use with onload

**Review by Ish451 — APPROVED** [review]
Looks Good, I like the variables created.

File: src/main.js [hunk 2]
```diff
@@ -8,12 +11,23 @@ var savedCovers = [
...
 
 // Create your event handlers and other functions here 👇
 
 
 // We've provided one function to get you started
+function displayMainPageCover() {
+  var randomCover = covers[getRandomIndex(covers)];
+  var randomTitle = titles[getRandomIndex(titles)];
+  var randomDescriptor = descriptors[getRandomIndex(descriptors)];
```
Comment (coopterrones) [inline src/main.js:23]: great use for avoiding generating the same descriptors twice.

File: src/main.js [hunk 3]
```diff
@@ -8,12 +11,23 @@ var savedCovers = [
...
+  var randomDescriptor2 = descriptors[getRandomIndex(descriptors)];
+  coverImage.setAttribute('src', randomCover);
+  coverTitle.innerHTML = randomTitle;
+  discriptorTagLine1.innerText = randomDescriptor;
```
Comment (coopterrones) [inline src/main.js:27]: We might consider using just one method here.

**Review by coopterrones — APPROVED** [review]

File: src/main.js (hunk 3 above)
Comment (mdflynn) [reply src/main.js:27]: Let's look into refining this.

**Pull request merged**